tests/Content/VersionTest.php

```diff
@@ -14,18 +15,78 @@ class VersionTest extends TestCase
 {
```

Comment:
The new testing setup is really great and very clean 👍

`Todo`: Could you please add tests for the following that are currently not covered:
- Different ways of retrieving language objects (suggestion: I'd add `@covers ::language` as a second covered method to relevant tests that test the different ways of passing/not passing a language).
- Test(s) for the `move()` behavior
- Touching all languages at once

Reply:
✅